src/workerd/api/tests/streams-r2-patterns-test.js

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2024 Cloudflare, Inc.
```

Comment:
Non-blocking observation: Similar to streams-respond-test, this only seems to port a few test cases.

Reply:
Yes, i'll follow up.